build-coordinator/src/main/java/org/jboss/pnc/core/builder/coordinator/BuildCoordinator.java

```diff
@@ -57,17 +59,17 @@
 @ApplicationScoped
 public class BuildCoordinator {
 
-    private Logger log = LoggerFactory.getLogger(BuildCoordinator.class);
+    private final Logger log = LoggerFactory.getLogger(BuildCoordinator.class);
 
     /**
      * Build tasks which are either waiting to be run or currently running.
      * The task is removed from the queue when the build is complete and the results
      * are stored to the database.
      */
-    private Queue<BuildTask> activeBuildTasks = new ConcurrentLinkedQueue<>(); //TODO garbage collector (time-out, error state)
+    private final Queue<BuildTask> activeBuildTasks = new ConcurrentLinkedQueue<>(); //TODO garbage collector (time-out, error state)
```

Comment:
What are we going to do with this queue? IMO - the easiest solution is to use JMS here.